cmd/account/mgmt/account-unassign.go

```diff
@@ -91,38 +86,40 @@ func (o *accountUnassignOptions) run() error {
 	}
 
 	o.awsClient = awsClient
+	var allUsers []string
 
 	if o.accountID != "" {
-
+		// Check aws tag to see if it's a ccs acct, if it's not return name of owner
 		accountUsername, err = o.checkForHiveNameTag(o.accountID)
 		if err != nil {
 			return err
 		}
-		accountIdList = append(accountIdList, o.accountID)
 
+		accountIdList = append(accountIdList, o.accountID)
 	}
 
 	if o.username != "" {
-		// Check that username is not a hive
+		// Check that username doesn't belong to a ccs acct
 		if strings.HasPrefix(o.username, "hive") {
 			return ErrHiveNameProvided
 		}
 
 		accountUsername = o.username
+
 		accountIdList, err = o.listAccountsFromUser(accountUsername)
 		if err != nil {
 			return err
 		}
 	}
 
+	allUsers = append(allUsers, accountUsername)
```

Comment:
One last critique and I think this will be done, I just ran through testing it on an account:

Let's remove this line and the whitespace line below (115 and 116).  The `allUsers` will be populated by the `listUsersFromAccount` function, where we get all the users inside an account.

The reason I ask is because for SRE accounts we might not have a `username` IAM User created in an account, which will throw a bunch of errors:

```
> ./osdctl account mgmt unassign -p osd-staging-1 -i 003857522824
Are you sure you want to unassign account(s) [[003857522824]] from kbater? [y/n] y
NoSuchEntity: The user with name kbater cannot be found.
        status code: 404, request id: 903f5f39-1bc7-4c32-bbfb-60612637f4ca
NoSuchEntity: The user with name kbater cannot be found.
        status code: 404, request id: 811d71d3-9561-45cb-9fb7-0a98f5ea1bd5
NoSuchEntity: The user with name kbater cannot be found.
        status code: 404, request id: bea51c94-830f-4911-bfaa-817c7472304a
NoSuchEntity: The user with name kbater cannot be found.
        status code: 404, request id: 73c800b1-0563-4293-8da3-0bf99db96ca0
NoSuchEntity: The user with name kbater cannot be found.
        status code: 404, request id: 513c4bf4-aa6a-4b3d-bf4b-85934fb7e11a
NoSuchEntity: The user with name kbater cannot be found.
        status code: 404, request id: 0a643040-9940-40f8-9dd7-207a7e7eebb9
NoSuchEntity: The user with name kbater cannot be found.
        status code: 404, request id: 2617842c-9c8d-46cf-9d17-1f4093f81f1b
NoSuchEntity: Login Profile for User anotheruser cannot be found.
        status code: 404, request id: 222b1d9f-cc7c-4a8b-b9e6-024026b7ef2e
user anotheruser successfully deleted
NoSuchEntity: Login Profile for User Auser cannot be found.
        status code: 404, request id: 5b13e499-4251-411a-89db-81d241234e92
user Auser successfully deleted
```

We should probably also supress the login profile errors as well, but I think we can circle back to that if it becomes a problem later.

So if we remove this line I think we'll be all set on this and this should be ready to ship :)